### PR TITLE
Add future downstream version to applicable files

### PR DESCRIPTION
**Pull request by mjivraja**
Please cherry-pick my commits into:

* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

**Review by Lennonka — CHANGES_REQUESTED** [review]
The link to the 6.15 upgrade guide has to be updated.
The guide was split into connected and disconnected, remember?

**Comment by mjivraja** [comment]
> The link to the 6.15 upgrade guide has to be updated. The guide was split into connected and disconnected, remember?

Yes, catch it. Thanks Lena 😅
Technical Writing isn't an easy job. 

I hope, you like my fix.

File: guides/common/attributes-satellite.adoc [hunk 1]
```diff
@@ -25,7 +25,8 @@
 :UpdatingDocURL: {BaseURL}updating_red_hat_satellite/index#
 :UpgradingDocURL: {BaseURL}upgrading_connected_red_hat_satellite_to_{ProjectVersion}/index#
 :UpgradingDisconnectedDocURL: {BaseURL}upgrading_disconnected_red_hat_satellite_to_{ProjectVersion}/index#
-:UpgradingPreviousDocURL: https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html-single/upgrading_red_hat_satellite_to_{ProjectVersionPrevious}/index#
+:UpgradingConnectedPreviousDocURL: https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html-single/upgrading_connected_red_hat_satellite_to_{ProjectVersionPrevious}/index#
+:UpgradingDisconnectedPreviousDocURL: https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html-single/upgrading_disconnected_red_hat_satellite_to_{ProjectVersionPrevious}/index#
```
Comment (Lennonka) [inline guides/common/attributes-satellite.adoc:29]: You can use the RHBaseURL now, can't you? :)
+ remove the word Connected from the attribute name for consistency

File: guides/common/attributes-titles.adoc [hunk 2]
```diff
@@ -27,7 +27,8 @@
 :UpdatingDocTitle: Updating {ProjectName}
 :UpgradingDocTitle: Upgrading {ProjectName} to {ProjectVersion}
 :UpgradingDisconnectedDocTitle: Upgrading disconnected {ProjectName} to {ProjectVersion}
-:UpgradingPreviousDocTitle: Upgrading {ProjectName} to {ProjectVersionPrevious}
+:UpgradingConnectedPreviousDocTitle: Upgrading connected {ProjectName} to {ProjectVersionPrevious}
```
Comment (Lennonka) [inline guides/common/attributes-titles.adoc:30]:
```suggestion
:UpgradingPreviousDocTitle: Upgrading connected {ProjectName} to {ProjectVersionPrevious}
```

**Review by Lennonka — CHANGES_REQUESTED** [review]

File: guides/common/attributes-satellite.adoc (hunk 1 above)
Comment (mjivraja) [reply guides/common/attributes-satellite.adoc]: Yes :+1:

**Review by Lennonka — APPROVED** [review]
I think it's all. Wouldn't mind another pair of eyes taking a look, I'm not in my best form today :D

**Comment by mjivraja** [comment]
> I think it's all. Wouldn't mind another pair of eyes taking a look, I'm not in my best form today :D

I hardly recall your form ever fluctuated. ♥️

**Pull request merged**